spec/index.bs

```diff
@@ -7381,35 +7380,36 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
     - {{GPUProgrammableStage}} |descriptor|
     - {{GPUPipelineLayout}} |layout|
 
-    Return `true` if all requirements in the following steps are satisfied, and `false` otherwise:
+    All of the requirements in the following steps |must| be met.
+    If they are met, return `true`; if not, return `false`.
```

Comment:
This feels stilted now, given how the other requirement is phrased. How about: 
```suggestion
    If any are unmet, return `false`; Otherwise return `true`.
```